openquake/qa_tests_data/event_based_risk/case_6c/expected/agg_curves_eb.csv

```diff
@@ -1,8 +1,21 @@
-#,,,,,,"generated_by='OpenQuake engine 3.9.0-gitc9a4f7bc4f', start_date='2020-03-05T07:08:25', checksum=2742411261, kind='agg_curves-rlzs', risk_investigation_time=1.0"
-return_period,rlz,loss_type,NAME_1,loss_value,loss_ratio,annual_frequency_of_exceedence
-50,0,structural,Region A,5.77686E+02,2.88843E-02,2.00000E-02
-50,0,structural,RegionB,5.74161E+02,1.14832E-02,2.00000E-02
-100,0,structural,Region A,7.73143E+02,3.86572E-02,1.00000E-02
-100,0,structural,RegionB,9.12244E+02,1.82449E-02,1.00000E-02
-200,0,structural,Region A,1.11514E+03,5.57572E-02,5.00000E-03
-200,0,structural,RegionB,2.85112E+03,5.70225E-02,5.00000E-03
+#,,,,,,"generated_by='OpenQuake engine 3.10.0-git2022d30ce1', start_date='2020-06-11T07:42:59', checksum=2658591021, kind='agg_curves-rlzs', risk_investigation_time=1.0"
+return_period,rlz,loss_type,site_id,loss_value,loss_ratio,annual_frequency_of_exceedence
+50,0,structural,1,1.71605E+02,1.71605E-02,2.00000E-02
+50,0,structural,2,1.05224E+02,1.05224E-02,2.00000E-02
+50,0,structural,3,1.16185E+02,1.16185E-02,2.00000E-02
+50,0,structural,4,2.26489E+02,2.26489E-02,2.00000E-02
+50,0,structural,5,2.19161E+02,2.19161E-02,2.00000E-02
+100,0,structural,0,1.42215E+02,1.42215E-02,1.00000E-02
+100,0,structural,1,3.89871E+02,3.89871E-02,1.00000E-02
+100,0,structural,2,1.15780E+02,1.15780E-02,1.00000E-02
+100,0,structural,3,1.47875E+02,1.47875E-02,1.00000E-02
+100,0,structural,4,5.66147E+02,5.66147E-02,1.00000E-02
+100,0,structural,5,5.53558E+02,5.53558E-02,1.00000E-02
+100,0,structural,6,1.44595E+02,INF,1.00000E-02
```

Comment:
The loss_ratios for `site_id = 6` are incorrect. All assets in this test case have a structural value of 10000.